docs/4.tutorials-and-examples/6.custom-webchat.mdx

```diff
@@ -13,51 +13,86 @@ This template shows how to create custom elements for webchat. All the custom co
 **src/webchat/index.js**
 
 ```javascript
-import Logo from '../assets/bot_vader.jpg'
-import { MyReply } from './myReply'
-
-export const theme = {
-  brandColor: '#ff0000',
-  textPlaceholder: 'Please, type your message:',
-  title: 'Custom Webchat',
-  brandIconUrl: Logo,
-  customReply: MyReply
+export const webchat = {
+  theme: {},
+
+  persistentMenu: [],
+
+  blockInputs: [],
+
+  // Webchat listeners
+  onInit: app => {},
+  onOpen: app => {},
+  onClose: app => {},
+  onMessage: app => {}
 }
 ```
 
-This is the main file of the webchat config. In this file, we export all the options that are going to be changed of the default webchat config.
+This is just a boilerplate code containing all the allowed definitions for your Webchat.
+- **theme**: In this object you can define the styles for your Webchat components.
+- **persistentMenu**: This will enable a persistent menu, which is a component that will help users to access the core functionalities of the bot at any point in the conversation.
```

Comment:
Optionally, a persistent menu can be accessed from a little button at the input text box. It allows the user to access popular functionalities at any point in the conversation.